linode/manager
# feat: [M3-7292] - Stepper component - Details content

## Description 📝
Stepper component - Details content.


| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/119517080/e2ce01a3-4147-44c0-8f90-d75465201d04) | ![image](https://github.com/linode/manager/assets/119517080/4f56021f-a68b-4767-a2fd-0c24aad2117f)|

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Checkout the branch and run the app locally
- Enable MSW


### Verification steps 
(How to verify changes)

- Navigate to http://localhost:3000/loadbalancers
- Click on "Create Load Balancer" Button.
- Verify ConfigurationDetails and its state management.
- Verify tests forConfigurationDetails.

## As an Author I have considered 🤔

*Check all that apply*
